WGS.config

```diff
@@ -157,8 +179,8 @@ process {
 
     withName: GATK_UnifiedGenotyper_BAF {
         cpus = 2
-        memory = '5G'
-        time = '2h'
+        memory = { 5.GB * task.attempt }
+        time = { 1.ms * bam_file.size() / 5300 * task.attempt }
```

Comment:
Are the resources of this process (GATK_UnifiedGenotyper_BAF) similar to GATK_UnifiedGenotyper_Fingerprint? Maybe the dynamic parameters resolve the differences.

Reply:
Nop `GATK_UnifiedGenotyper_BAF` is a bigger target and therefor requires more resources.